### DISCUSSION TITLE
posts/002-diet-campaign/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#7] [Announcements]
# posts/002-diet-campaign/

減肥能讓俺的車更省油．更重要的是，俺的前女友 BMI 只有 16，她的車大概可以直接起飛，，，

**Comment by PermeationLoop** [comment]
➗生學校飯堂活全家，要啥沒啥，學生天天高強度嗯造碳水，教科書級別的本子飲食風範，，，

人活着不就是要有盼頭嗎
想到明天中午的學校飯堂，甚至沒有我週末做的盒飯有盼頭
唉明天中午吃麥當勞漢堡王算了

**Comment by erikopak-cmyk** [comment]
會計攝入熱量，每天有熱量缺口就ok了
